pysparkling/_version.py

```diff
@@ -10,12 +10,14 @@
 
 """Git implementation of _version.py."""
 
-import errno
```

Comment:
This I also don't understand exactly. The only reason is that errno is a binary package, and the others are within `Lib`. I don't see an option to change this ordering though.

I could do it manually, but I would like to enable this automatic option as it saved me already many headaches because of unused imports blocking builds...